src/components/button/primaryButton.jsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * Created by jonlazarini on 14/02/17.
+ */
+import React from 'react';
+import Button from './index.jsx';
+import styled from 'styled-components';
+
+
+// Style wrapper for default buttons - Composition
+const PrimaryWrapper = (Comp) => styled(Comp)`
```

Comment:
(Comp) fails the linter. Overall make sure the linter passes before submitting a pull request.